### PR TITLE
[doc] Update operator installation doc

### PR DESCRIPTION
**Pull request by yhwang**
Refine the operator installation doc
- also define KFDEF_URL for IKS
- add indention
- only use yq v4+

Signed-off-by: Yihong Wang <yh.wang@ibm.com>

### CODE REVIEW

File: docs/mlx-install-operator.md [hunk 1]
```diff
@@ -2,51 +2,54 @@
...
+    ```
 
 3. (Optional) Enable development mode
-When developing with MLX, we recommend to run the below command to disable monitoring from the operator framework
-```shell
-kubectl patch deployment kubeflow-operator -n operators -p '{"spec":{"replicas":0}}'
-```
+    When developing with MLX, we recommend to run the below command to disable monitoring from the operator framework
+    ```shell
+    kubectl patch deployment kubeflow-operator -n operators -p '{"spec":{"replicas":0}}'
```
Comment (yhwang) [inline docs/mlx-install-operator.md:52]: one question about this step. is this still valid? I can't find `deploy/kubeflow-operator` under `operators` ns.

Comment (Tomcli) [reply docs/mlx-install-operator.md:69]: oh it should be `deploy/mlx-operator` because we renamed it above.

Comment (yhwang) [reply docs/mlx-install-operator.md]: got it and updated it to `mlx-operator`. I also found the `sed` command that replaces `kubeflow-operator` to `mlx-operator` accidentally replaces the image url as well. fixed it too.

**Review by ckadner — APPROVED** [review]
LGTM

Thanks @yhwang

**Pull request merged**